paymentsheet/src/test/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowControllerTest.kt

```diff
@@ -1384,3 +1255,31 @@ internal class DefaultFlowControllerTest {
         private val STATUS_BAR_COLOR = Color.GREEN
     }
 }
+
+private suspend fun PaymentSheet.FlowController.configureExpectingSuccess(
+    clientSecret: String = PaymentSheetFixtures.CLIENT_SECRET,
+    configuration: PaymentSheet.Configuration? = null,
+) {
+    val configureResult = CompletableDeferred<Throwable?>()
```

Comment:
Do we get any timeout with `CompletableDeferred`? We could use a `Turbine` instead.